recipe/meta.yaml

```diff
@@ -1,43 +1,53 @@
-{% set version = "0.2.3" %}
 {% set name = "intake-parquet" %}
-{% set sha256 = "353078f8b3358abdf0b1fa959fb8bd7f40466c8d04c474f51015b13822b9b8e3" %}
+{% set version = "0.3.0" %}
 
 package:
-   name: {{ name }}
-   version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/intake-parquet-{{ version }}.tar.gz
+  sha256: 57e43795f53d5e44d375683f13532bae248005ff428d177613bc7607ce4a0f89
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  skip: true # [py<37 or s390x]
+  script: 
+    # Remove the out of date versioneer.py file from the source for py3.12 issue
+    - rm -f versioneer.py  # [py>311]
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
+    - wheel
+    - versioneer ==0.29
   run:
-    - python >=3.5
-    - intake >=0.3
+    - python
+    - intake
     - pandas
     - fastparquet
     - pyarrow
-    - dask
-    - jinja2
 
 test:
   imports:
     - intake_parquet
+  commands:
+    - pip check
+  requires:
+    - pip
```

Comment:
Upstream tests pls 🙏 https://github.com/intake/intake-parquet/blob/f7a83a2e561fca1de8fcbc0ff06bb8ae04937116/conda/meta.yaml#L26

Reply:
Done! Took some encouragement and one is completely out of date, hard to tell as they don't run CI/CD as far as I can see at the moment